src/aomodule.h

```diff
@@ -65,7 +65,7 @@ It can either be called as a member function of an AudioDevice object:\n\
 or as a standalone function which takes the integer id or the string name of the\n\
 driver:\n\
    ao.driver_info(1) or ao.driver_info(\"pulse\")\n\
-If None is passed or the argument omited a list of informations for all drivers\n\
+If None is passed (or the argument omited) information for all drivers\n\
```

Comment:
You might still to state that a `list` is returned instead of a `dict`.

Also I just noticed, that `omitted` contains a typo as well, but I like the readability now.

Reply:
Done!